lib/ontologies_linked_data/models/agents/identifier.rb

```diff
@@ -25,10 +25,10 @@ def embedded_doc
         "#{self.id.split('/').last}"
       end
 
-      def no_url(inst,attr)
+      def valid_orcid_ror(inst,attr)
         inst.bring(attr) if inst.bring?(attr)
         notation = inst.send(attr)
-        return  notation&.start_with?('http') ? [:no_url, "`notation` must not be a URL"]  : []
+        return  notation&.start_with?('invalid') ? ["The indentifier you entered is invalid"]  : []
```

Comment:
this is not valid, first from where will the term `invalid`  come, and validator functions need to return an array with as the first value the error key, and the second value a text to describe the error

Reply:
invalid comes from the UI when the orcid || ror are not valid